### DISCUSSION TITLE
Increase BG requirements for more even games

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#5360] [Suggestions]
Currently, bgs are doomed by lots of people quitting (or dc'ing) or heavily imbalanced by selected classes accidentally (and repetitively) being stacked in the same team.

If battlegrounds (specifically the max level bgs) require at least 20 players (for warsong, ab, eots) to start, individual leavers have less weight and the odds of balanced teams are higher given larger teams.
Further, we would be more blizzlike and have more battlegrounds the way they are intended.

**Comment by Eledor-collab** [comment]
I perfectly agree with the fact that bgs nowadays a heavily affected by quitting. Most of my friends won't queue bgs because of the uneven number of members per team that usually happens after one team starts to win.

"You just got capped, someone quits, the other team receives one additional player, and we didn't even reach the middle of the match", they say.

So, another way to "partly" address this issue would be to severely penalize desertion. Quitting behavior, so far, doesn't really bear any real consequences.  Perhaps extending the duration of the deserter debuff to 1 day and applying it to all characters on the same account, or another type of measure that would make people think twice before leaving its team to an slow and on-sided doomed match, could reduce the number of games that gives pvp a bad reputation to newcomers or casual pvp players.
